kbdcfg.lua

```diff
@@ -59,6 +64,18 @@ function kbdcfg.add_additional_layout(name, label, subcmd)
     table.insert(kbdcfg.additional_layouts, layout)
 end
 
+function kbd_client_update(c)
+    if not c then
+        return
+    end
+
+    if not c.kbd_layout then
+        c.kbd_layout = kbdcfg.layouts[1].name
```

Comment:
May be it is better to use here `kbdcfg.current` instead of constant 1. In this case we will set current layout for every new windows.